packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -15,7 +15,7 @@ type Path =
 export const gasRequiredTo: GasRequiredTo = {
   deployInfrastructureContracts: {
     vanillaNitro: {
-      NitroAdjudicator: 2421626, // Singleton
+      NitroAdjudicator: 2421830, // Singleton
```

Comment:
This is what you had just referenced? This is pretty surprising.

If the reported differences are true, and not the result of weird gas estimation approximations leaving the exact reporting (rather than +/- 1%) will present some learning opportunities. How much value there is in these opportunities is another question.

Reply:
I don't think this is quite what I just mentioned in standup. This is an expected increase, I think, since we may be doing more work by checking a non-strict inequality (rather than a strict one). 

I suppose we could save 4 gas by converting it back into a strict inequality.